test/spec/runners/bls.ex

```diff
@@ -76,8 +76,14 @@ defmodule BLSTestRunner do
     assert false
   end
 
-  defp assert_fast_aggregate_verify(_input, _output) do
-    assert false
+  defp assert_fast_aggregate_verify(
+         %{message: message, pubkeys: pubkeys, signature: signature},
+         output
+       ) do
+    case Bls.fast_aggregate_verify(pubkeys, message, signature) do
+      {:ok, confirmation} -> assert confirmation == output
+      {result, _error_msg} -> assert result == :error
```

Comment:
We should add the error message as the assert's message, to improve debuggability:
```suggestion
      {result, error_msg} -> assert result == :error, error_msg
```